hassio/dashboard/hassio-hass-update.html

```diff
@@ -35,10 +36,14 @@
               </template>
             </div>
             <div class='card-actions'>
+              <a
+                href='https://github.com/home-assistant/home-assistant/releases'
```

Comment:
maybe we should point to correct version

Reply:
Prefer to just point at least at https://home-assistant.io/blog/ , preferable point at the right url.

Reply:
GH release note list all breaking changes, even if you skipped some releases. Blog url has to be fetched from somewhere